data/musicalbums.json

```diff
@@ -0,0 +1,25 @@
+{
+  "musicalbums": [
+    {
+      "json_class": "Book",
+      "published_date": "0001-01-01",
+      "id": "c3e43dea-392a-471b-bfb9-e89b5486a708",
+      "archived": true,
+      "cover_state": "bad"
+    },
+    {
```

Comment:
When you save the music album it shouldn't show that is a book and also it doesn't have the correct data inside. Check it, the cover_state is a book's propriety.

Reply:
@marurevi Thanks for the feedback, however i dont think storing of data is required in this part of the project.

![image](https://user-images.githubusercontent.com/97225349/182624071-871b98b5-42d0-4775-ada8-5eb6b20a7c9e.png)
Please help confirm.